meinberlin/apps/dashboard/templates/meinberlin_dashboard/includes/preview.html

```diff
@@ -0,0 +1,12 @@
+{% load i18n extproject_tags %}
```

Comment:
the file has to be moved to "meinberlin/templates/meinberlin_dashboard2/includes/preview.html"